runner/actions.go

```diff
@@ -32,6 +33,12 @@ func actionRunner(c *cli.Context) error {
 	isMetricDisabled := c.Bool("disable-metrics")
 
 	ArtifactAddr := c.String("artifact-addr")
+	ClusterID := os.Getenv("CLUSTER_ID")
+	if ClusterID == "" {
+		logrus.Error("Failed to setup runner: CLUSTER_ID environment variable not provided")
+		return nil
+	}
```

Comment:
Can we avoid this? Like use the default clusterName if `CLUSTER_ID` isn't provided

Reply:
Also why are the variable names capitalised?

Reply:
the cluster-id is provided through env variable during setup itself.
eg. if we run the following cmd:   ` space-cli setup ` , the cluster-name would be default and cluster-id would be default--some-gibberish. cluster-id makes it uniquely identifiable

Reply:
The user can provide his own cluster_id as well right?